### PR TITLE
Add Underline in Account Header

### PR DESCRIPTION
**Pull request by Crazypkr1099**

### CODE REVIEW

**Comment by Teprifer** [comment]
Hrm, the lines might sit better with a few tweaks, as they are they're a bit dominating.

The lines seems a bit too bold - note lines elsewhere are fairly faint.

Since they aren't sections of the sidebar, but rather subsections of all accounts, it might make more sense for them to not go fully across and instead only start where the text does, and end at the end of the numbers.

File: packages/desktop-client/src/components/sidebar/Accounts.tsx [hunk 1]
```diff
@@ -149,7 +159,7 @@ export function Accounts({
         }}
         onClick={onAddAccount}
         Icon={SvgAdd}
-        title="Add account"
+        title="Add Account"
```
[the remark this answers is not in the document]
Comment (Crazypkr1099) [reply packages/desktop-client/src/components/sidebar/Accounts.tsx:162]: Agreed. I'll fix that.

**Comment by Teprifer** [comment]
Thinner line is good, thought I'd mock up my poorly worded other suggestion about line width, please forgive my bad paint job skills.

I think the second does look better and gives a better sense of hierarchy than all the way across since there's already a fully across line break between the menu options and the accounts section.

image

**Comment by Crazypkr1099** [comment]
> Thinner line is good, thought I'd mock up my poorly worded other suggestion about line width, please forgive my bad paint job skills.
> 
> I think the second does look better and gives a better sense of hierarchy than all the way across since there's already a fully across line break between the menu options and the accounts section.
> 
> image

I understand what you are saying, but to achieve this is quite difficult as the current way it is coded. I think its such a minor detail its not a huge deal right now to have the line extend the entire way. If you select the account, it adds the small rectangle showing its selected. If we have the line only part of the way, it would look very strange. Right now it looks normal.

Screenshot 2024-06-07 195155


Thanks. I would like to merge this as is.

**Comment by carkom** [comment]
> Thinner line is good, thought I'd mock up my poorly worded other suggestion about line width, please forgive my bad paint job skills.
> 
> I think the second does look better and gives a better sense of hierarchy than all the way across since there's already a fully across line break between the menu options and the accounts section.
> 
> image
> 

I agree that the second is more desireable. With lines all the way across it looks like entirely new sections of the menu rather than a subsection of accounts. While it may be simpler to code the first, the second is doable and I think we should make an effort to use that look instead.

**Comment by Crazypkr1099** [comment]
> > Thinner line is good, thought I'd mock up my poorly worded other suggestion about line width, please forgive my bad paint job skills.
> > I think the second does look better and gives a better sense of hierarchy than all the way across since there's already a fully across line break between the menu options and the accounts section.
> > image
> 
> I agree that the second is more desireable. With lines all the way across it looks like entirely new sections of the menu rather than a subsection of accounts. While it may be simpler to code the first, the second is doable and I think we should make an effort to use that look instead.

Kk, I managed to make it not take up the entire width... but when you select it, the border doesn't take the highlighted color. I am not skilled enough in JS/React to figure out how to do this, so if someone could take over from here that would be great (or pass me in the right direction). Thanks :)

**Comment by youngcw** [comment]
> Kk, I managed to make it not take up the entire width... but when you select it, the border doesn't take the highlighted color.

Im not sure what you are referring to by this.  It looks good to me, pretty much exactly what I think @carkom was asking for.

**Comment by youngcw** [comment]
@Teprifer @carkom Any complaints remaining? or are we good?

**Comment by Teprifer** [comment]
Looks good to me, thanks for taking the feedback on board @Crazypkr1099 :)

**Comment by Crazypkr1099** [comment]
> Looks good to me, thanks for taking the feedback on board @Crazypkr1099 :)

Can we merge it then?

**Review by youngcw — APPROVED** [review]

File: packages/desktop-client/src/components/sidebar/Account.tsx [hunk 2]
```diff
@@ -148,6 +148,12 @@ export function Account({
             </View>
 
             <AlignedText
+              style={
+                (name === 'Off budget' || name === 'For budget') && {
+                  borderBottom: `1.5px solid rgba(255,255,255,0.4)`,
+                  paddingBottom: '3px;',
```
Comment (lelemm) [inline packages/desktop-client/src/components/sidebar/Account.tsx:154]: After updating some branches with master, there is a console log all the time:

image

**Review by lelemm — COMMENTED** [review]
Found this minor problem in the master branch after updating my branches to master

File: packages/desktop-client/src/components/sidebar/Account.tsx (hunk 2 above)
Comment (Crazypkr1099) [reply packages/desktop-client/src/components/sidebar/Account.tsx:154]: Ah crap sorry... Didn't know about this. I'll create a PR tonight to fix this.

Comment (Crazypkr1099) [reply packages/desktop-client/src/components/sidebar/Account.tsx:154]: #2881

**Pull request merged**